backend/PostProcessing/PostProcessingClasses/ImmediateArithmeticOperation.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#ImmediateArithmeticOperation.py
+#-------------------------------
+# Created By : Matthew Kastl
+#-------------------------------
+""" This file is a postprocessing class under the IPostProcessing interface.
+The post processing in this file preforms an arithmetic operation on columns in a data frame by a constant.
+By the index of the data frame!
+ """ 
+#-------------------------------
+# 
+#
+#Imports
+from PostProcessing.IPostProcessing import IPostProcessing
+from pandas import DataFrame
+
+
+class ImmediateArithmeticOperation(IPostProcessing):
+
+    def post_process(self, data: DataFrame, op: str, left_col_key: str, value: float, out_col_key: str) -> DataFrame:
+        """The post processing in this file preforms an arithmetic operation on columns in a data frame by a constant.
+        By the index of the data frame!
+
+        Args:
+            data DataFrame: The DataFrame containing this collation of the data that has been collected.
+            op - the operation to preform (ex. addition)
+            left_col_key - The key the column that will be on the left side of the operation.
+            right_col_key - The key for the column on the right side of the operation.
```

Comment:
the function defintion doesn't have a right_col_key parameter, it has a value: float parameter